src/index.css

```diff
@@ -27,6 +27,8 @@ html {
      * .monaco-editor .minimap { z-index: 5; }
      * ```
      */
+    --ylv-panel-title-button-z-index: 0;
+    --ylv-panel-query-input-z-index: 0;
```

Comment:
If in https://github.com/y-scope/yscope-log-viewer/pull/107/files#r1841517108 and https://github.com/y-scope/yscope-log-viewer/pull/107/files#r1841516945 we decide that we do not need the z-index overrides any more, we shall also remove these.